controllers/tenant/storageclass.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	csiprovisionerv1alpha1 "github.com/kubermatic/kubevirt-csi-driver-operator/api/v1alpha1"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
Do we really need `v1` import alias here? Let's have `corev1` if we have to do alias. 😅